### ISSUE TITLE
get() and filter() in trackingcategories not working

### ISSUE THREAD

**Issue by AmidoriA** [freakboy3742/pyxero#73]
I have test with my code below, it working fine.
```
self.xero.trackingcategories.all()
```

However, when I try to filter it. it not working regardless which parameter i put in (always return None)
```
# all of these not working
self.xero.trackingcategories.get(u'f3a8bc1d-xxxx-xxxx-xxxx-ea3d696eac94')
self.xero.trackingcategories.get('f3a8bc1d-xxxx-xxxx-xxxx-ea3d696eac94')
self.xero.trackingcategories.filter(Name='Units')
self.xero.trackingcategories.filter(Name=u'Units')
```

Tested on Public Application

**Comment by AmidoriA** [comment]
It seem the whole trackingcategory is have something wrong with xml structure

example PUT from the API
```
<TrackingCategories>
    <TrackingCategory>
      <Name>Things</Name>
    </TrackingCategory>
</TrackingCategories>
```

generated from PyXero
```
<TrackingCategorie>
    <Status>ACTIVE</Status>
    <Name>Things</Name>
    <Options>
        <Option>
            <Status>ACTIVE</Status>
            <Name>a</Name>
        </Option>
    </Options>
</TrackingCategorie>
```

as you can see, the structure seem not right. The ```<TrackingCategory>``` haven't been put in.

I will try to fix it myself and see if it works and share with you guys. If anyone have any suggestion, please let me know

**Comment by aidanlister** [comment]
I believe this is fixed in 0.7.0:

This is what I see with xero.trackingcategories.all():

```
[{'Name': 'Region',
  'Options': [{'HasValidationErrors': False,
               'IsActive': True,
               'IsArchived': False,
               'IsDeleted': False,
               'Name': 'Eastside',
               'Status': 'ACTIVE',
               'TrackingOptionID': 'ae777a87-5ef3-4fa0-a4f0-d10e1f13073a'},
              {'HasValidationErrors': False,
               'IsActive': True,
               'IsArchived': False,
               'IsDeleted': False,
               'Name': 'North',
               'Status': 'ACTIVE',
               'TrackingOptionID': '3f05cdf9-246b-46a2-bf6f-441da1b09b89'},
              {'HasValidationErrors': False,
               'IsActive': True,
               'IsArchived': False,
               'IsDeleted': False,
               'Name': 'South',
               'Status': 'ACTIVE',
               'TrackingOptionID': '9f93bb09-3c8c-4d43-a6fe-862772d77bd9'},
              {'HasValidationErrors': False,
               'IsActive': True,
               'IsArchived': False,
               'IsDeleted': False,
               'Name': 'West Coast',
               'Status': 'ACTIVE',
               'TrackingOptionID': '2ba6a2af-11cb-452d-9867-dce88add6856'}],
  'Status': 'ACTIVE',
  'TrackingCategoryID': '093af706-c2aa-4d97-a4ce-2d205a017eac'},
 {'Name': 'Widget',
  'Options': [{'HasValidationErrors': False,
               'IsActive': True,
               'IsArchived': False,
               'IsDeleted': False,
               'Name': 'Bar',
               'Status': 'ACTIVE',
               'TrackingOptionID': 'e797ed26-2718-4006-9403-d1215de2433b'},
              {'HasValidationErrors': False,
               'IsActive': True,
               'IsArchived': False,
               'IsDeleted': False,
               'Name': 'Baz',
               'Status': 'ACTIVE',
               'TrackingOptionID': '00fe3878-7982-4c55-80c2-3e37947280b1'},
              {'HasValidationErrors': False,
               'IsActive': True,
               'IsArchived': False,
               'IsDeleted': False,
               'Name': 'Foo',
               'Status': 'ACTIVE',
               'TrackingOptionID': '5315f297-b15f-445f-9de1-1011234c3103'}],
  'Status': 'ACTIVE',
  'TrackingCategoryID': '82c872bb-0720-4551-9df9-f227433311ca'}]
```

This is what I see with xero.trackingcategories.filter(Name='Region'):

```
[{'Name': 'Region',
  'Options': [{'HasValidationErrors': False,
               'IsActive': True,
               'IsArchived': False,
               'IsDeleted': False,
               'Name': 'Eastside',
               'Status': 'ACTIVE',
               'TrackingOptionID': 'ae777a87-5ef3-4fa0-a4f0-d10e1f13073a'},
              {'HasValidationErrors': False,
               'IsActive': True,
               'IsArchived': False,
               'IsDeleted': False,
               'Name': 'North',
               'Status': 'ACTIVE',
               'TrackingOptionID': '3f05cdf9-246b-46a2-bf6f-441da1b09b89'},
              {'HasValidationErrors': False,
               'IsActive': True,
               'IsArchived': False,
               'IsDeleted': False,
               'Name': 'South',
               'Status': 'ACTIVE',
               'TrackingOptionID': '9f93bb09-3c8c-4d43-a6fe-862772d77bd9'},
              {'HasValidationErrors': False,
               'IsActive': True,
               'IsArchived': False,
               'IsDeleted': False,
               'Name': 'West Coast',
               'Status': 'ACTIVE',
               'TrackingOptionID': '2ba6a2af-11cb-452d-9867-dce88add6856'}],
  'Status': 'ACTIVE',
  'TrackingCategoryID': '093af706-c2aa-4d97-a4ce-2d205a017eac'}]
```
